src/app/layouts/PageLayout/ChangelogOverlay/ChangelogOverlay.data.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+interface ChangelogItem {
+  version?: string;
+  data: Array<JSX.Element>;
+}
+
+export interface ChangelogRenderItem {
+  data: {
+    Component: () => JSX.Element;
+  }[];
```

Comment:
you could do this without wrapping `ChangelogItem.data` into components.  
`ChangelogRenderItem` is redundant